doc/changes.md

```diff
@@ -1,5 +1,89 @@
 # Changelog
 
+## Release v0.9.0 (2023/03/31)
+
+[![Digital Object Identifier for PyGMT v0.9.0](https://zenodo.org/badge/DOI/10.5281/zenodo.7772533.svg)](https://doi.org/10.5281/zenodo.7772533)
+
+### Highlights
+
+* 🎉 **Ninth minor release of PyGMT** 🎉
+* Add Figure.tilemap to plot XYZ tile maps ([#2394](https://github.com/GenericMappingTools/pygmt/pull/2394))
+* Add function to load raster tile maps using contextily ([#2125](https://github.com/GenericMappingTools/pygmt/pull/2125))
```

Comment:
Any other highlights? We could also add something like "9 new gallery + inline examples"?

Reply:
Good idea! I added your suggestion below. Please feel welcome to commit any changes (as could others) - I'm off to sleep for the night but will revisit this in the morning.